icu4c/source/i18n/vtzone.cpp

```diff
@@ -1217,6 +1217,13 @@ VTimeZone::getOffset(UDate date, UBool local, int32_t& rawOffset,
     return tz->getOffset(date, local, rawOffset, dstOffset, status);
 }
 
+void VTimeZone::getOffsetFromLocal(
```

Comment:
add newline after void

Reply:
why?

Reply:
There are so many C++ function in ICU start that way I thought that is the coding style in ICU.